homeassistant/components/otbr/websocket_api.py

```diff
@@ -58,6 +59,7 @@ async def websocket_info(
         {
             "url": data.url,
             "active_dataset_tlvs": dataset_tlvs.hex() if dataset_tlvs else None,
+            "border_agent_id": border_agent_id.hex(),
```

Comment:
Afaik an older OTBR could have no border ID, so maybe we need to handle None here too? :thinking:

Reply:
Handled in https://github.com/home-assistant/core/pull/98395